spring-data-cassandra/src/test/java/org/springframework/data/cassandra/util/SpelUtilsTest.java

```diff
@@ -0,0 +1,117 @@
+
+import static org.junit.Assert.*;
+
+import javax.swing.text.Segment;
+
+import org.junit.Test;
+import org.springframework.expression.EvaluationContext;
+import org.springframework.expression.ExpressionParser;
+import org.springframework.expression.spel.SpelCompilerMode;
+import org.springframework.expression.spel.SpelParserConfiguration;
+import org.springframework.expression.spel.standard.SpelExpressionParser;
+import org.springframework.expression.spel.support.StandardEvaluationContext;
+
+/**
+ * Unit tests for class {@link SpelUtils org.springframework.data.cassandra.util.SpelUtils}.
+ *
+ * @author Michael Hausegger
+ * @date 20.02.2017
+ * @see SpelUtils
+ **/
+public class SpelUtilsTest {
+
+
+	@Test
+	public void testOne() throws Exception {
+
+		SpelExpressionParser spelExpressionParser = SpelUtils.DEFAULT_PARSER;
+		String string = SpelUtils.evaluate((CharSequence) "cellEditor", (EvaluationContext) null, (ExpressionParser) spelExpressionParser);
+
+		assertEquals("cellEditor", string);
+	}
+
+
+	@Test
+	public void testTwo() throws Exception {
+
+		SpelExpressionParser spelExpressionParser = new SpelExpressionParser();
+		String string = SpelUtils.evaluate((CharSequence) "", (EvaluationContext) null, (ExpressionParser) spelExpressionParser);
+
+		assertEquals("", string);
+	}
+
+
+	@Test
+	public void testThree() throws Exception {
+
+		Class<String> clasz = String.class;
+		SpelParserConfiguration spelParserConfiguration = new SpelParserConfiguration(false, false);
+		SpelExpressionParser spelExpressionParser = new SpelExpressionParser(spelParserConfiguration);
+		String string = SpelUtils.evaluate((CharSequence) "+18:00", (EvaluationContext) null, clasz, (ExpressionParser) spelExpressionParser);
+
+		assertFalse(spelParserConfiguration.isAutoGrowNullReferences());
+		assertFalse(spelParserConfiguration.isAutoGrowCollections());
+
+		assertEquals(SpelCompilerMode.OFF, spelParserConfiguration.getCompilerMode());
+		assertEquals(2147483647, spelParserConfiguration.getMaximumAutoGrowSize());
+
+		assertEquals("+18:00", string);
+	}
+
+
+	@Test
+	public void testFour() throws Exception {
+
+		Class<Object> clasz = Object.class;
+		String string = (String) SpelUtils.evaluate((CharSequence) "caret-begin-word", (EvaluationContext) null, clasz);
+
+		assertEquals("caret-begin-word", string);
+	}
+
+
+	@Test
+	public void testFive() throws Exception {
+
+		String string = SpelUtils.evaluate((CharSequence) "", (EvaluationContext) null);
+
+		assertEquals("", string);
+	}
+
+
+	@Test
+	public void testSix() throws Exception {
+
+		String string = SpelUtils.evaluate((CharSequence) "caret-begin-word", (EvaluationContext) null);
+
+		assertEquals("caret-begin-word", string);
+	}
+
+
+	@Test(expected = StringIndexOutOfBoundsException.class)
+	public void testSevenRaisesStringIndexOutOfBoundsException() throws Exception {
+
+		char[] charArray = new char[2];
+		Segment segment = new Segment(charArray, (-2826), 0);
```

Comment:
Please don't use Swing types in here. `StringBuilder` might be the better choice.